config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -1644,6 +1644,7 @@ spec:
                         type: array
                         x-kubernetes-list-map-keys:
                         - containerPort
+                        - protocol
```

Comment:
```suggestion
```
Due to a bug in the generator tool this should be removed, otherwise the kubernetes api server will not allow to install this to a k8s cluster v 1.18+